tests/types/test_numpy.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+from psycopg.adapt import PyFormat
+from psycopg.types.numpy import register_numpy
+
+pytest.importorskip("numpy")
```

Comment:
There should be a workflow in CI to run these tests too, but not the main one because numpy shouldn't be a mandatory dependency. See dns/postgis tests:

https://github.com/psycopg/psycopg/blob/85c965089ae9cd7234dcced357b6e9b4a0637b6d/.github/workflows/tests.yml#L37

https://github.com/psycopg/psycopg/blob/85c965089ae9cd7234dcced357b6e9b4a0637b6d/.github/workflows/tests.yml#L94-L98

https://github.com/psycopg/psycopg/blob/85c965089ae9cd7234dcced357b6e9b4a0637b6d/tests/types/test_shapely.py#L13

https://github.com/psycopg/psycopg/blob/85c965089ae9cd7234dcced357b6e9b4a0637b6d/psycopg/tox.ini#L31-L33

Reply:
Numpy workflow is up and running and passes the test. I still trying to figure it out how to skip a test when numpy is not installed. Looking at `test_shapely`,  my assumption was that `pytest.importskyp` was responsible for skipping the test in case the import failed. But I can't reproduce the same behavior in `test_numpy`. Will figure it out though